build_android/src/main/res/values-es/strings.xml

```diff
@@ -0,0 +1,183 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+	<!-- Application Navigation -->
+
+	<string name="about_play">Acerca de Play!</string>
+	<string name="about_play_emu">Play! Emulador de PlayStation 2</string>
```

Comment:
here is should `Play! PlayStation 2 Emulator`

Reply:
Ok, done. So this string shouldn't be translated... 🤔